userspace/libsinsp/sinsp.h

```diff
@@ -1205,7 +1205,7 @@ VISIBILITY_PRIVATE
 	std::vector<sinsp_protodecoder*> m_decoders_reset_list;
 
 	//
-	// meta event management for other sources like k8s, mesos.
+	// meta event management for other sources.
 	//
 	sinsp_evt* m_metaevt;
```

Comment:
Out of curiosity, what are remaining usages of `m_metaevt` now?

Reply:
this is a good question, i have no idea :confounded:

Reply:
It's now only in place `get_procs_cpu_from_driver`. I'd love to investigate turning that into an async event queue and call it a day.